packages/core/src/document.ts

```diff
@@ -384,14 +384,14 @@ class Document extends EventEmitter {
   /**
    * Given two different DocStates representing two different conflicting histories of the same
    * document, pick which commit to accept, in accordance with our conflict resolution strategy
-   * @param state1 - first log's state
-   * @param state2 - second log's state
-   * @returns true if state2's log should be taken, or false if state1's log should be taken
```

Comment:
Hmm... So I very intentionally didn't include any reference to 'local' or 'remote' when I wrote this, as an earlier version of this logic always preferred the remote log in the case of a tie and it's important that this behavior be entirely deterministic. So I think it's important that this function be written in a way that it is just comparing two random equally valid logs and not have any information about which came from where.

I do agree, however, that the name of the function and it returning a bool makes it hard to know from reading calling code what it's supposed to be doing.  I think perhaps the thing to do is change this to instead of returning a bool, return the DocState that it chose.  Feel free to do that here, or file a ticket and I can take care of it in a separate commit.  Either way though, can we revert the change to the function and variable names?